server/server_test.go

```diff
@@ -559,6 +559,43 @@ var _ = Describe("Server package test", func() {
 					testURL("PUT", url, memberTokenID, testResource, http.StatusUnauthorized)
 				})
 			})
+
+			It("should not create resource in different tenant than relation", func() {
```

Comment:
Since these test are basically the same, I suggest using `DescribeTable`.

Reply:
i don't like that idea however i might create Context and move common part to BeforeEach